source/wp-content/themes/wporg-news-2021/block-template-parts/content-category-community.html

```diff
@@ -1,10 +1,10 @@
 <!-- wp:group {"tagName":"header","className":"entry-header",style":{"spacing":{"padding":{"top":"30px","right":"30px","bottom":"30px","left":"30px"}}}} -->
 <header class="wp-block-group entry-header">
-	<!-- wp:post-title {"level":3,"isLink":true} /-->
 	<!-- wp:post-featured-image {"isLink":true,"width":"200","height":"200"} /-->
 
 	<!-- wp:group {"className":"entry-meta"} -->
 	<div class="wp-block-group entry-meta">
+		<!-- wp:post-title {"level":3,"isLink":true} /-->
```

Comment:
The parent heading here is `h1`, so this should be an `h2` to avoid skipping levels ([more info on why here](https://webaim.org/techniques/semanticstructure/#headings)).

<img width="310" alt="Screen Shot 2022-01-17 at 12 44 53 PM" src="https://user-images.githubusercontent.com/541093/149816970-857f6f1b-45b1-4b95-b956-9441a70e0d01.png">